src/Toolkit/Toolkit.WinUI/UI/Controls/Legend/Legend.Theme.xaml

```diff
@@ -1,12 +1,12 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+﻿<ResourceDictionary x:Class="Esri.ArcGISRuntime.Toolkit.LegendResources"
+                    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="using:Esri.ArcGISRuntime.Toolkit"
                     xmlns:mapping="using:Esri.ArcGISRuntime.Mapping"
                     xmlns:controls="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+                    xmlns:ui="using:Esri.ArcGISRuntime.Toolkit.UI"
                     xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
```

Comment:
Aside: We can get rid of this old contract check, since our minimum Windows requirement is now 10.0.19041 which shipped with UniversalApiContract 10. Doesn't have to be this PR, just writing down a thought here.